doc/user/content/concepts/clusters.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Clusters"
+description: "Clusters"
+menu:
+  main:
+    parent: 'concepts'
+    weight: 5
+    identifier: 'concepts-clusters'
+---
+
+## Overview
```

Comment:
Except for adding headings and updating links to new concepts pages, the content is more or less the same.